src/common/testing/test_utils/container_runner.cc

```diff
@@ -46,7 +46,7 @@ ContainerRunner::ContainerRunner(std::filesystem::path image_tar,
   std::vector<std::string_view> lines = absl::StrSplit(out, "\n", absl::SkipWhitespace());
   CHECK(!lines.empty());
   std::string_view image_line = lines.back();
-  constexpr std::string_view kLoadedImagePrefix = "Loaded image(s): ";
+  constexpr std::string_view kLoadedImagePrefix = "Loaded image";
   CHECK(absl::StartsWith(image_line, kLoadedImagePrefix));
   image_line.remove_prefix(kLoadedImagePrefix.length());
```

Comment:
This breaks the remove prefix. We probably should either regex or split on `:`

Reply:
See also https://github.com/containers/podman/commit/5023ce585c724b0c2bd49945393585d72b6e6342